app/domain/authentication/validate_security.rb

```diff
@@ -1,15 +1,77 @@
 # frozen_string_literal: true
 
+require 'authentication/webservices'
+
 module Authentication
   ValidateSecurity = CommandClass.new(
     dependencies: {
-      security: ::Authentication::Security.new
+      role_class: ::Authentication::MemoizedRole,
+      webservice_resource_class: ::Resource
     },
-    inputs:       %i(input enabled_authenticators)
+    inputs: %i(webservice account user_id enabled_authenticators)
   ) do
 
     def call
-      @security.validate(@input.to_access_request(@enabled_authenticators))
+      # No checks required for default conjur authn
+      return if default_conjur_authn?
+
+      validate_webservice_is_whitelisted
+      validate_webservice_exist
+      validate_user_is_defined
+      validate_user_has_access
+    end
+
+    private
+
+    def default_conjur_authn?
+      @webservice.authenticator_name ==
+        ::Authentication::Common.default_authenticator_name
+    end
+
+    def validate_webservice_exist
```

Comment:
english grammar nit.  should be: `validate_webservice_exists`

Reply:
correct :)